### DISCUSSION TITLE
classes/class_audiostreamplaybacksynchronized

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#258] [User-contributed notes]
# classes/class_audiostreamplaybacksynchronized

Inherits: AudioStreamPlayback< RefCounted< Object There is currently no description for this class. Please help us by contributing one!

https://docs.godotengine.org/en/4.3/classes/class_audiostreamplaybacksynchronized.html

**Comment by GeminiSquishGames** [comment]
I feel like this is how you are supposed to access the playback when you have several AudioStreamSync streams that are AudioStreamGenerator, but since I don't have reference, I don't know what methods to use for polyphonic synthesis except to instantiate one AudioStreamPlayer and AudioStreamGenerator for each note of polyphony. 

It would seem more efficient (maybe...) to have multiple generators similar playback in an AudioStreamSync setup, or maybe something like it as mixing polyphony down to frames or in a buffered playback is a lot of CPU work. We could use more documentation for this, AudioStreamPlaybackResampled, and AudioStreamPlaybackOggVorbis. There are probably others I'm forgetting about but I notice these have been untouched for a while and the knowledge would be good.
